incubator/vault/templates/service.yaml

```diff
@@ -27,6 +27,10 @@ spec:
     protocol: TCP
     targetPort: {{ .Values.service.port }}
     name: api
+  - port: {{ .Values.service.clusterExternalPort }}
```

Comment:
Should this port always be exposed or should it be flagged behind an if block?

Reply:
I added a conditional block to preserve the original design of this service. When a value for the `service.clusterExternalPort` is provided, the cluster port will be exposed in the service.